internal/transform/scope-css.go

```diff
@@ -122,8 +124,9 @@ outer:
 						isBracket = true
 						isElement = false
 						isPseudoState = false
-						// if there is no selector before an attribute selector, then assume "*"
-						if n == 0 {
+
+						// if there is no selector before an attribute selector and we're not in a delcaration, assume "*"
+						if n == 0 && declaration == "" {
```

Comment:
Previously, this logic was designed to handle attribute selectors `[data-click]` but it _also_ unintentionally caught cases like:
```
grid-template-columns: [content-start] 1fr [content-end];
```

This fixes that by checking if we've already encountered a declaration.